sklearn/model_selection/_search.py

```diff
@@ -367,13 +367,12 @@ def fit_grid_point(X, y, estimator, parameters, train, test, scorer,
     # NOTE we are not using the return value as the scorer by itself should be
     # validated before. We use check_scoring only to reject multimetric scorer
     check_scoring(estimator, scorer)
-    scores, n_samples_test = _fit_and_score(estimator, X, y,
-                                            scorer, train,
-                                            test, verbose, parameters,
-                                            fit_params=fit_params,
-                                            return_n_test_samples=True,
-                                            error_score=error_score)
-    return scores, parameters, n_samples_test
+    results = _fit_and_score(estimator, X, y, scorer, train,
+                             test, verbose, parameters,
+                             fit_params=fit_params,
+                             return_n_test_samples=True,
+                             error_score=error_score)
+    return results["test_scores"], parameters, results["n_test_samples"]
```

Comment:
Would it make sense to get the parameters from the dict?

```suggestion
    return results["test_scores"], results["parameters"], results["n_test_samples"]
```

Reply:
In this case, I would say no. `return_parameters` is set to false so `parameters` is not in the dict and `fit_grid_point` is deprecated.

Reply:
OK make sense